ros/api/v1/hosts.py

```diff
@@ -524,7 +525,11 @@ def get(self):
             PerformanceProfile.report_date <= stale_date
         ).count()
 
-        non_psi_count = systems_with_performance_record_queryset.filter_by(psi_enabled=False).count()
+        non_psi_count = systems_with_performance_record_queryset.filter(
+            PerformanceProfile.operating_system['major'].astext.cast(Integer) != 7,
```

Comment:
Does it make sense to keep psi_enabled False by default? and hence for RHEL 7 it will be False and we won't need this extra `!= 7` condition?